File: trunk/unityClientSample/Assets/src/protocol/RoundRequest.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// roundOver 协议:25
    /// </summary>
	[Proto(value=25,description="roundOver")]
	[ProtoContract]
	public class RoundRequest
	{
        /// <summary>
        ///  roomId
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string roomId;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/battle/RoundRequest.java
package org.ngame.protocol.battle;

import lombok.Data;
import org.ngame.protocol.annotation.Proto;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = 25, description = "roundOver")
public class RoundRequest
{
	@Protobuf(description = "roomId")
	private String roomId;
}
